### DISCUSSION TITLE
MSA_Transformer pretrained weights

### DISCUSSION THREAD

**Discussion by hypnopump** [facebookresearch/esm#36] [General] [answered]
Is there any plan to make the model available in torch hub? If it already is, what's the url to load if from?

**Comment by joshim5** [comment, marked as answer]
Yes, we just added torch hub support in commit [e96b12d74](https://github.com/facebookresearch/esm/commit/e96b12d740230c4765b5deebb29845c3098231eb). 

Additionally, the model weights can be found here: https://dl.fbaipublicfiles.com/fair-esm/models/esm_msa1_t12_100M_UR50S.pt

**Reply by tomsercu** [reply]
ah yes -- see #40  :)
